packages/patternfly-4/react-core/src/components/DataList/DataListItemCells.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+
+export interface DataListItemContentProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the DataList item Content Wrapper.  Children should be one ore more <DataListCell> nodes */
+  className?: string;
+  /** Array of <DataListCell> nodes that are rendered one after the other. */
+  dataListCells?: React.ReactNode[];
```

Comment:
This can just be `React.ReactNode` since the definition looks like this:
```
interface ReactNodeArray extends Array<ReactNode> {}
type ReactFragment = {} | ReactNodeArray;
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```